DEVELOPMENT.rst

```diff
@@ -84,13 +73,25 @@ When debugging something email related change the email backend to console::
 
     $ python manage.py start --debug-email
 
-To start OpenSlides with Daphne run::
+e. Setup and start the client
+'''''''''''''''''''''''''''''
 
-    $ python manage.py runserver
+Go in the client's directory in a second command-line interface::
+
+    $ cd client/
+
+Install all dependencies and start the development server::
+
+    $ npm install  # TODO: yarn install gave me errors a few weeks ago..
+    $ yarn start
 
-Use gulp watch in a second command-line interface::
+Now the client is available under ``localhost:4200``.
 
-    $ node_modules/.bin/gulp watch
+If you do not need to work with the client, you can build the client and let it be delivered by the server directly::
+
+    $ yarn build # TODO: this is not tested!
```

Comment:
Please remove the TODO. @FinnStutzenstein Can you test this please?

Reply:
I tested it. It works fine

Reply:
removed